WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -38,14 +38,20 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v18.9-whats-new"
+msgctxt "v19.0-whats-new"
 msgid ""
-"Made a New Year’s resolution to publish more content on your site? We made a few updates to the Block Editor to help you do just that.\n"
+"What time is it? Our brand-new time zone selection screen knows. Get a suggested time zone based on your device and improve search while you’re at it.\n"
 "\n"
-"- When you’re editing Paragraph blocks, you can highlight specific text in any color you’d like.\n"
-"- Brand-new Gallery blocks will default to the new design. You can also auto-convert any old galleries to the new format without a hitch.\n"
-"- The editor now auto-scrolls, so you can always see what you’re typing. No more disappearing text, no more guesswork.\n"
-"- We squashed a bug where the background color wasn’t showing up for pre-formatted blocks on sites that use standard themes.\n"
+"If you’re using a free plan, video uploads are now limited to five minutes per file. Don’t worry, that won’t affect any videos you’ve already uploaded to your site.\n"
+"\n"
+"We’ve also made some changes to the block editor:\n"
+"\n"
+"- We solved translation issues that caused some user interface elements (like text alignment information and the block editor modal window) to only appear in English.\n"
+"- We added language translation for Jetpack and Layout Grid blocks.\n"
+"- We fixed a bug that would reset text formatting to default after hitting backspace. Bold on, friends.\n"
+"- Multi-line block titles are now center-aligned in the inserter menu.\n"
+"\n"
+"Finally, the “Share WordPress with a friend” row is back where it belongs on the Me screen. Reunited and it feels so good.\n"
```

Comment:
I don't think the new tag for https://github.com/wordpress-mobile/docker-glotpress-linter/pull/1 got created by @jkmassel yet (nor applied to WPiOS), so I think our po linter will still not run `xgettext` and thus miss that syntax error…

```suggestion
"Finally, the \“Share WordPress with a friend\” row is back where it belongs on the Me screen. Reunited and it feels so good.\n"
```

Reply:
Huh, so strangely, running `xgettext -o - WordPress/Jetpack/Resources/AppStoreStrings.po` on that branch does not seem to trigger any warning, and does seem to be happy with the quotes not being escaped… 🤔  I'm confused… (as similar cases of unescaped quotes clearly triggered `xgettext` on some previous release notes)

Reply:
Ohhhh wait, I just saw that those are not U+0022 straight quotes, but instead curved quotes!

That explains why, if I try locally to add those backslash escapes in front of them (like I suggest above) and run `xgettext` afterwards, I get:
```
$ xgettext -o - WordPress/Resources/AppStoreStrings.po
WordPress/Resources/AppStoreStrings.po:50:16: invalid control sequence
WordPress/Resources/AppStoreStrings.po:50:47: invalid control sequence
xgettext: found 2 fatal errors
```

Because I am trying to backslash-escape curved quotes (U+201C and U+201D), not regular ones. Because they are curved quotes, they **don't** need escaping after all!
My bad, forget about theGitHub  suggestion then! 🙃